tests/e2e/pubsub/pubsub_test.go

```diff
@@ -74,7 +74,7 @@ func sendToPublisher(t *testing.T, publisherExternalURL string, topic string) ([
 
 		statusCode, err := postSingleMessage(url, jsonValue)
 		// return on an unsuccessful publish
-		if statusCode != http.StatusOK {
+		if statusCode != http.StatusNoContent {
```

Comment:
@wcs1only  check both??

Reply:
There is no path in the code which returns 200 OK anymore.